### DISCUSSION TITLE
Bookmarklet

### DISCUSSION THREAD

**Discussion by Samq64** [aidan-crockett/view-images#4] [Show and tell]
Someone said that going to https://gosoccerboy5.github.io/view-images and pasting the URL into the text box was to inconvenient, so I decided to bookmarklet so people can get the link without leaving the page or opening a new tab!
___
Here it is:
`(function(){let a=prompt('Paste image URL:');if(a!==null){location.assign('https://gosoccerboy5.github.io/view-images/#'+a);}})();`
Unfortunately I can't turn it into a hyperlink because it interferes with the markdown. I was also hoping to make it copy the link to the clipboard automatically, but I don't think bookmarklets are capable of that.

Anyway, I hope someone will find this useful!

**Comment by aidan-crockett** [comment]
ok

**Reply by aidan-crockett** [reply]
you could also use [window.location.assign](https://developer.mozilla.org/en-US/docs/Web/API/Location/assign)

**Reply by Samq64** [reply]
> you could also use [window.location.assign](https://developer.mozilla.org/en-US/docs/Web/API/Location/assign)

I just updated the bookmarklet. Thanks for this tip!
